events/community-meeting.md

```diff
@@ -63,9 +63,25 @@ Also, if you are doing a live coding demo, please make sure it has a reasonable
 - Ensure you are presenting from a quiet environment.
 - If you run out of time while performing your demo, you may ask the audience if they would like a follow-up at a subsequent meeting. If there is enthusiastic support, the community team will help schedule a continuation.
 
-## SIG Updates 
-
-SIGs will give a community update at least once per release cycle per the [schedule](https://docs.google.com/spreadsheets/d/1adztrJ05mQ_cjatYSnvyiy85KjuI6-GuXsRsP-T2R3k). 
+## SIG Updates
+
+SIGs will give a community update at least once per release cycle per the [schedule](https://docs.google.com/spreadsheets/d/1adztrJ05mQ_cjatYSnvyiy85KjuI6-GuXsRsP-T2R3k).
+The SIG Update should mention:
+
+- Topics where input is being sought from other SIGs
+- Topics that could affect other SIGs
+- Currently active themes and goals in the SIG
+  - Broad description of future themes and goals if possible
```

Comment:
Was this intentionally indented?  It makes me wonder if this big bullet list might be organized into a couple three or four major topic areas with the detailed bullets under them?  But that's really a nit and can be modified in future if desired.

/lgtm